python-api/gstswitch/testsource.py

```diff
@@ -333,6 +331,30 @@ class VideoSrc(object):
     """
     HOST = '127.0.0.1'
 
+    PATTERN_SMPTE = 0
+    PATTERN_SNOW = 1
+    PATTERN_BLACK = 2
+    PATTERN_WHITE = 3
+    PATTERN_RED = 4
+    PATTERN_GREEN = 5
+    PATTERN_BLUE = 6
+    PATTERN_CHECKERS1 = 7
+    PATTERN_CHECKERS2 = 8
+    PATTERN_CHECKERS4 = 9
+    PATTERN_CHECKERS8 = 10
+    PATTERN_CIRCULAR = 11
+    PATTERN_BLINK = 12
+    PATTERN_SMPTE75 = 13
+    PATTERN_ZONE_PLATE = 14
+    PATTERN_GAMUT = 15
+    PATTERN_CHROMA_ZONE_PLATE = 16
+    PATTERN_SOLID_COLOR = 17
+    PATTERN_BALL = 18
+    PATTERN_SMPTE100 = 19
+    PATTERN_BAR = 20
+    PATTERN_PINWHEEL = 21
+    PATTERN_SPOKES = 22
```

Comment:
Feel free not to, but this might be worth using the enum module for -> https://pypi.python.org/pypi/enum/0.4.4